fdbserver/OldTLogServer_6_0.actor.cpp

```diff
@@ -1158,7 +1158,7 @@ Version poppedVersion(Reference<LogData> self, Tag tag) {
 		if (tag == txsTag || tag.locality == tagLocalityTxs) {
 			return 0;
 		}
-		return self->recoveredAt;
+		return self->recoveredAt + 1;
```

Comment:
I do not know anything about this, but if this is from OldTLogServer, I would feel the bug should be in the master branch too? Do we want to fix master and then merge back to ptxn?

Reply:
This is a cherry-pick from master, which Steve fixed.